addons/main.go

```diff
@@ -96,6 +98,16 @@ func main() {
 
 	ctx := ctrl.SetupSignalHandler()
 
+	webhooktls = &webhook.TLSCredentials{
+		CAName:       "self.signed.cert",
+		Organization: "tanzu.vmware",
+		Hostname:     "hostname",
+		DnsNames:     []string{"webhook-service", "webhook-service.default", "webhook-service.default.svc"},
+		ValidFrom:    time.Now(),
+		MaxAge:       time.Hour * 24 * 365,
+	}
+	webhooktls.GetServerRSAKeyPEM()
+
```

Comment:
This whole section does not belong here. It might be better to move it to the SetupWebhookWithManager function of each one of the webhook definitions.

Reply:
Take this out and instead add tests to addons/webhook/certificates_test.go